numpyro/contrib/module.py

```diff
@@ -85,6 +85,8 @@ def flax_module(
         # feed in dummy data to init params
         args = (jnp.ones(input_shape),) if input_shape is not None else args
         rng_key = numpyro.prng_key()
+        if rng_key is None:
+            rng_key = random.key(0)
```

Comment:
I had to do it this way since `rng_key = numpyro.prng_key() or random.key(0)` we giving an error

```python
E       ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```